### ISSUE TITLE
add VaultRole.Ensure support

### ISSUE THREAD

**Issue by xh3b4sd** [giantswarm/vaultrole#2]
See https://github.com/giantswarm/cluster-service/pull/198#discussion_r148730544.

**Comment by puja108** [comment]
Yes, i feel Ensure is important here, as it's not always about creating a new role but ensuring there's the right roles there.
